_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,220 @@
+    transactions).
+
+    Participants in the mailing list discussion seemed supportive of the
+    proposal.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **nix-bitcoin**: nixbitcoindev appeared on the Stephan Livera Podcast to
+  discuss nix-bitcoin.  nix-bitcoin is an experimental but promising project
+  that aims to improve the security of installing Bitcoin and Lightning nodes. It
+  uses the functional Linux distribution NixOS where upgrading is equivalent to
+  installing from scratch due to atomicity. Through minimalism, reproducibility,
+  compartmentalization and defense in depth nix-bitcoin aims to reduce the attack
+  surface of a Bitcoin and Lightning software stack with programs unaware of
+  other running processes unless they need to be. ([transcript][nixbitcoin
+  transcript], [audio][nixbitcoin audio])
+
+- **Taproot activation**: Eric Lombrozo, Luke Dashjr and Aaron van Wirdum
+  discussed the alternative [Taproot][topic taproot] activation proposals (see
+  [Newsletter #107][news107 taproot activation]) and gave their opinions in a
+  personal capacity on what lessons, if any, could be derived from activating the
+  SegWit soft fork.  Lombrozo and Dashjr both believe that the Taproot activation
+  process should not be drawn out and that all opposition, criticism or review of
+  the proposed code changes should be flushed out prior to starting any possible
+  activation process. As a result they support a single phase BIP 8 activation
+  method with parameters still to be determined. Community feedback on activation
+  proposals continues to be collected. ([transcript][activation transcript],
+  [video][activation video])
+
+- **Signet**: Kalle Alm and AJ Towns participated in a discussion on
+  [Signet][topic signet].  The design decisions of Signet were explored as well
+  as the mechanics of how testnet and regtest work. For more details see the
+  [_Default signet discussion_ news item](#default-signet-discussion).
+  ([transcript][signet transcript], [video][signet video])
+
+- **Bitcoin Core GUI meeting**: Anonymized participants including designers and
+  developers met to discuss the Bitcoin Core graphical user interface, its
+  current state, how it could be improved or revamped and the constraints that
+  any possible revamp would be subject to. For example, one change that has been
+  discussed in the past is moving from Qt Widgets to the Qt QML framework which
+  is more flexible and easily customized. ([transcript][bitcoin core gui
+  transcript]).
+
+- **Sydney meetup discussion** Ruben Somsen and Rusty Russell participated in a
+  discussion on statechains, upgrading LN channels and [lnprototest][]. Somsen
+  outlined how statechains (see [Newsletter #100][news100 statechains]) could
```

Comment:
This is actually Newsletter 91 not Newsletter 100. Apologies, this was my error. https://bitcoinops.org/en/newsletters/2020/04/01/#implementing-statechains-without-schnorr-or-eltoo

Reply:
Two replacements of s/100/91